aws/resource_aws_autoscaling_group_test.go

```diff
@@ -916,75 +916,6 @@ func TestAccAWSAutoScalingGroup_TargetGroupArns(t *testing.T) {
 	})
 }
 
-func testAccAWSAutoScalingGroupConfig_TargetGroupArns(rName string, tgCount int) string {
```

Comment:
This isn't removed just moved down to be with the other config instead of randomly in the middle of tests... 😃